packages/stack.rb

```diff
@@ -2,22 +2,30 @@
 
 class Stack < Package
   description 'The Haskell Tool Stack - Stack is a cross-platform program for developing Haskell projects. It is aimed at Haskellers both new and experienced.'
-  homepage 'https://docs.haskellstack.org/en/stable/README/'
+  homepage 'https://docs.haskellstack.org/'
   @_ver = '2.5.1'
   version @_ver
```

Comment:
Not sure why the version wasn't incremented here.

Reply:
The old package works fine, a build from source will throw errors. It didn't seem necessary at the time to bump the version however if you'd like a version bump I can put up a PR for that.